crypt-port.h

```diff
@@ -256,4 +256,22 @@ void _xcrypt_secure_memset (void *s, size_t len)
 #define gensalt_sha_rn           _crypt_gensalt_sha_rn
 #endif
 
+#if INCLUDE_yescrypt
+#define PBKDF2_SHA256            _crypt_PBKDF2_SHA256
+#define yescrypt_digest_shared   _crypt_yescrypt_digest_shared
```

Comment:
If dropping ROM support, it's also fairly easy and non-invasive to drop `yescrypt_digest_shared`.